components/arena/docker/arena_launcher.py

```diff
@@ -249,7 +249,10 @@ def main(argv=None):
   parser.add_argument('--tensorboard-image', type=str, default='tensorflow/tensorflow:1.12.0')
   parser.add_argument('--timeout-hours', type=int,
                       default=200,
-                      help='Time in hours to wait for the Job submitted by arena to complete')
+                      help='Time in minutes to wait for the Job submitted by arena to complete')
```

Comment:
It should be hours, right?

Reply:
Yes, because training always takes more than 1 hour.